app/views/dynamic_pages/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="page-title">
+  <div class="row">
+    <div class="col-9">
+      <h1> BJC Pages </h1>
+    </div>
+
+    <div class="col-3">
+      <% if session[:user_id] && Teacher.find(session[:user_id]).admin %>
```

Comment:
For the dynamic_page#index view, I changed this to `logged_in? && current_user.admin?` since `current_user` will return nil if no one is logged in. For the other pages that the public cannot access, I changed it to just `current_user.admin?`